README.md

```diff
@@ -1,6 +1,22 @@
 # aider-environment
 
-run
+## note about docker
+
+If you are not familliar with docker, is better before running script from this repository ensure, that you have configured and running docker. Try run command:
+```
+$ docker ps
+CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
+```
+If you are got message as above, all is good and you can continue to section [run](#run).
```

Comment:
@srbt how about:

If you see a message similar to the one above, everything is set up correctly and you can proceed to the [Run]() section.